__tests__/app.test.js

```diff
@@ -146,4 +146,50 @@ describe("/api/articles", () => {
         });
     });
   });
+  describe("/api/articles", () => {
+    describe("GET", () => {
+      test("GET 200: responds with 200 status", () => {
+        return request(app).get("/api/articles").expect(200);
+      });
+      test("GET 200: responds with array of all articles, length 13", () => {
```

Comment:
There would be nothing wrong with combining these tests. In integration testing we're looking for the full picture of what the server would respond with from the endpoint you're visiting.